### PR TITLE
feat: support refresh materialized view

### PR DESCRIPTION
**Pull request by QuentinJanuel**
Hi, this PR adds the functionality to refresh materialized views!

### CODE REVIEW

**Comment by QuentinJanuel** [comment]
@koskimas @igalklebanov Sorry am I missing anything regarding the contributions guidelines? Please tell me if there is any issue with this PR so that I can fix it

**Comment by igalklebanov** [comment]
Hey 👋 

Sorry for the delay, IRL stuff. Overall, it looks good. Let's get those tests 🟢.

**Comment by QuentinJanuel** [comment]
Okay I think I fixed the tests, they were not passing because I was trying to drop a materialized view without specifying `.materialized()`.
I cannot run the tests locally though so I'm sorry if we're still not done yet 😕

**Comment by QuentinJanuel** [comment]
Sorry for taking so much time. There were still one failing test left, but this time hopefully everything will pass 🤞

File: src/operation-node/refresh-view-node.ts
Comment (koskimas) [inline src/operation-node/refresh-view-node.ts:1]: Rename to `refresh-materialized-view-node`

**Review by koskimas — APPROVED** [review]
I rebased this to v0.28. You need to export the node from the `index.ts` file to make the tests on that branch pass. Then this is good to go.

File: src/operation-node/refresh-view-node.ts
Comment (QuentinJanuel) [reply src/operation-node/refresh-view-node.ts:1]: Thank you! Should be good to go now then!

**Pull request merged**